### PR TITLE
use new urls module

### PR DESCRIPTION
**Pull request by snopoke**
`django.conf.urls.defaults` is deprecated and removed in Django 1.6

### CODE REVIEW

File: corehq/apps/groups/urls.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-#from django.conf.urls.defaults import patterns, url
-from django.conf.urls.defaults import *
+#from django.conf.urls import patterns, url
```
Comment (dimagimon) [inline corehq/apps/groups/urls.py]: E265 block comment should start with '# '

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by snopoke** [comment]
build failure fixed by https://github.com/dimagi/commcare-hq/pull/5272

**Comment by czue** [comment]
looks good pending test rerun

**Pull request merged**